presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -27,22 +21,23 @@ services:
       - '10000:10000'
       - '50070:50070'
       - '50075:50075'
-  
-  presto-master-common:
+
+  presto-master:
     extends:
       service: java-8-base
     hostname: presto-master
     volumes:
       - ../../../../presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
```

Comment:
why do we need `tempto-configuration` on `presto-master`? Can't we keep it on `application-runner`?